### ISSUE TITLE
Update practice text to be more accurate

### ISSUE THREAD

**Issue by Bialogs** [mitre-cyber-academy/mitre-cyber-academy.github.io#25]
"Practice online at any time with the Online Practice Range, host your own range, create your own challenges, or visit our GitHub to test your hacking skills in a series of challenges."

Reword this to remove online range.

**Comment by rbclark** [comment]
To add to that, should we remove the block stating "create your own challenges"? To me that implies we provide some sort of challenge sort of framework to help people with development and we really don't have anything like that.

**Comment by Bialogs** [comment]
You created some instructions for creating challenges in 2013 or 2014, cant remember, but would that be useful enough to keep?

**Comment by rbclark** [comment]
If we were to add something like that then I could see justifying keeping that text however otherwise it is somewhat confusing.

**Comment by Bialogs** [comment]
https://github.com/mitre-cyber-academy/how-to-write-a-challenge
